deploy/charts/secret-manager/templates/deployment.yaml

```diff
@@ -11,12 +11,15 @@ spec:
       {{- include "secret-manager.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-    {{- with .Values.podAnnotations }}
+      {{- if .Values.podAnnotations }}
```

Comment:
Is this difference just syntax or does this change the generated manifests?

Reply:
No, it doesn't change the syntax. I'll revert that.